pkgs/applications/misc/nemu/default.nix

```diff
@@ -0,0 +1,107 @@
+{ stdenv, config, lib, fetchFromGitHub
+, cmake
+, pkg-config
+, gettext
+, libpthreadstubs
+, libudev
+, libusb1
+, ncurses
+, qemu
+, sqlite
+
+, libxml2
+, libarchive
+, graphviz
+, dbus
+
+, debugBuild ? false
+, ovfSupport ? true
+, saveVMSupport ? false  # requires patched version of QEMU, override them all if needed
+, withVNCClientSupport ? true
+, withSpice ? true
+, withNetworkMap ? false
+, withDbus ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nemu";
+  version = "2.6.0";
+
+  src = fetchFromGitHub {
+    owner = "nemuTUI";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0wbx2g7nizczrsrv4xaqm7ma1dd7m35dkhjcgczzh1z14gdpz4gw";
+  };
+
+  system.requiredKernelConfig = with config.lib.kernelConfig; [
+    (isEnabled "VETH")
+    (isEnabled "MACVTAP")
+  ];
+
+  nativeBuildInputs = [ cmake pkg-config ];
+
+  buildInputs = [
+    gettext
+    libarchive
+    libpthreadstubs
+    libudev
+    libusb1
+    libxml2
+    ncurses
+    qemu
+    sqlite
+  ] ++ lib.optional withNetworkMap graphviz
+    ++ lib.optional withDbus dbus;
+
+  cmakeFlags = lib.optional debugBuild "-DNM_DEBUG=ON"
+    ++ lib.optional ovfSupport "-DNM_WITH_OVF_SUPPORT=ON"
+    ++ lib.optional saveVMSupport "-DNM_SAVEVM_SNAPSHOTS=ON"
+    ++ lib.optional withVNCClientSupport "-DNM_WITH_VNC_CLIENT=ON"
+    ++ lib.optional withSpice "-DNM_WITH_SPICE=ON"
+    ++ lib.optional withNetworkMap "-DNM_WITH_NETWORK_MAP=ON"
+    ++ lib.optional withDbus "-DNM_WITH_DBUS=ON";
+
+  preConfigure = ''
+    patchShebangs .
+    substituteInPlace CMakeLists.txt --replace 'USR_PREFIX "/usr"' "USR_PREFIX \"$(out)\""
```

Comment:
I see upstream has improved a bit their usage of CMake, but this is still present, it'd be nice to try to contribute upstream a more non-FHS friendly change for their `CMakeLists.txt`.